test/libsolidity/syntaxTests/inlineAssembly/solidity_keywords.sol

```diff
@@ -0,0 +1,113 @@
+contract C {
+  function f() view public {
+    assembly {
+  // These are keywords of Solidity -- a copy from liblangutil/Token.h.
```

Comment:
Not a big deal but indentation is off :p

Reply:
I know because it is a copy of the `yulSyntaxTest` and i had no patience reformatting it.

Reply:
vim: `ggvG=`

Reply:
No success. Feel free to push over the commit.